src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -608,20 +607,10 @@ public Page<CaseNote> getOffenderCaseNotes(@PathVariable("offenderNo") @Paramete
                                                @RequestParam(value = "type", required = false) @Parameter(description = "Filter by case note type", example = "GEN") final String type,
                                                @RequestParam(value = "subType", required = false) @Parameter(description = "Filter by case note sub-type", example = "OBS") final String subType,
                                                @RequestParam(value = "prisonId", required = false) @Parameter(description = "Filter by the ID of the prison", example = "LEI") final String prisonId,
+                                               @RequestParam(value = "caseNoteTypeSubTypes", required = false) @Parameter(description = "Filter by list of case note types and optional case note sub types separated by plus", example = "KA+KE,OBS,POMK+GEN") final List<String> caseNoteTypeSubTypes,
```

Comment:
I think this would be better without the `caseNote` prefix on the parameter - maybe `typesSubTypes` to indicate that multiple types can be provided too.

Reply:
renamed to `typesSubTypes`